records/generator_test.go

```diff
@@ -164,6 +169,7 @@ func testRecordGenerator(t *testing.T, spec labels.Func, ipSources []string) Rec
 	if err != nil {
 		t.Fatal(err)
 	} else if err = json.Unmarshal(b, &sj); err != nil {
+		t.Log("err: ", err)
```

Comment:
this is strange: we're already logging the error w/ t.Fatal(), why the additional t.Log()?